package/kedro_viz/server.py

```diff
@@ -452,10 +448,15 @@ def format_pipeline_data(
                 edges_list.append(edge)
             namespace_tags[namespace].update(node.tags)
             node_dependencies[task_id].add(namespace_id)
-
+    # Parameters and data
     for namespace, tag_names in sorted(namespace_tags.items()):
         is_param = bool("param" in namespace.lower())
         node_id = _hash(namespace)
+        if is_param:
+            _NODES[node_id] = {"type": "parameters", "obj": None}
+        else:
+            _get_dataset_node(node_id, namespace)
```

Comment:
Can we make the API symmetric? It's not a good idea for a "getter" to have side effects.
```python
node_data = {"type": "parameters", "obj": None} if is_param else _get_dataset_node(node_id, namespace)
_NODES[node_id] = node_data
```